nixos/modules/services/web-apps/miniflux.nix

```diff
@@ -103,45 +105,27 @@ in
         ++ lib.optionals cfg.createDatabaseLocally [ "postgresql.service" "miniflux-dbsetup.service" ];
```

Comment:
The upstream `miniflux.service` already sets

```
After=network.target postgresql.service
```
https://github.com/miniflux/v2/blob/v2.2.0/packaging/systemd/miniflux.service#L12

`After=` is, to my best knowledge, not something you can reset/override, only append to.
Meaning, even if you try something like `after = [ "" "network.target" ]`, it will still depend on `postgresql.service`.

This renders the conditional partially useless.